crates/aptos/src/node/local_testnet/node.rs

```diff
@@ -155,7 +155,12 @@ impl NodeManager {
 
         // So long as the indexer relies on storage indexing tables, this must be set
         // for the indexer GRPC stream on the node to work.
-        node_config.storage.enable_indexer = run_txn_stream;
+        // node_config.storage.enable_indexer = run_txn_stream;
```

Comment:
let's remove commented out code lines
```suggestion
        // node_config.storage.enable_indexer = run_txn_stream;
```